config/main.py

```diff
@@ -1714,6 +1715,21 @@ def synchronous_mode(sync_mode):
     else:
         raise click.BadParameter("Error: Invalid argument %s, expect either enable or disable" % sync_mode)
 
+#
+# 'yang_config_validation' command ('config yang_config_validation ...')
+# 
+@config.command('yang_config_validation')
+@click.argument('yang_config_validation', metavar='<enable|disable>', required=True)
+def yang_config_validation(yang_config_validation):
```

Comment:
>yang_config_validation [](http://example.com/codeflow?start=4&length=22)

Let's focus on `config yang_config_validation` in this PR, and move the sample implementation of yang validation to another PR.